flow-libs/cssnano.js.flow

```diff
@@ -1,13 +1,14 @@
 // @flow strict-local
 
-// Derived from the source of cssnano located at
+// Modified from the source of cssnano located at
 // https://github.com/cssnano/cssnano and its types at
 // https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cssnano/index.d.ts
 
 declare module 'cssnano' {
+  import type { Plugin } from "postcss"
   declare interface CSSNanoOptions {
     configFile?: string;
     preset?: [string, mixed] | string;
   }
-  declare module.exports: (?CSSNanoOptions) => void;
+  declare module.exports: (?CSSNanoOptions) => Plugin;
```

Comment:
This was based on this:
https://github.com/cssnano/cssnano/blob/39b4cc089e8825d308df1fa8a7313d38d8f1f2f7/packages/example-cli-usage/postcss.config.js#L2